### ISSUE TITLE
Assembling Machine won't accept circuits insert with conduit and electum pipes

### ISSUE THREAD

**Issue by Axiflare** [GTNewHorizons/GT-New-Horizons-Modpack#3170]
#### Which modpack version are you using?

2.0.4.4.

#### What did you try to do, and what did you expect to happen?

Request a lapotron crystal from my AE system.

#### What happened instead? (Attach screenshots if needed)

Recipe stalled on raw lap crystal and HV circuits, assembler will not accept 2/3 of the HV tier circuits via item pipes since updating to 2.0.4.4, even though lots of recipes exist. Putting the items in manually still works, though. I went on further to test all circuits up to LuV to possibly find a pattern. 

Affected by this bug are:

Microprocessor (LV)
Processor Assembly (HV)
Nanoprocessor (HV)

Circuits not affected: MVs, EVs, IVs, LuVs.

Hope this information helps!

**Comment by Dream-Master** [comment]
will check

**Comment by noobyaran9** [comment]
I think its because of the NBT data.

**Comment by Dream-Master** [comment]
No so far i know the circuits not have nbt data only data sticks or orbs.

**Comment by Dream-Master** [comment]
Works well in sp. Not sure why mp is different.

**Comment by Dream-Master** [comment]
@Axiflare Are all recipes shown or only one?

**Comment by Axiflare** [comment]
Edited for clarification, sorry about that!

**Comment by Dream-Master** [comment]
Not tested it via pipes.

**Comment by lexx1191** [comment]
similarly, do not craft  lapotron crystal from my AE system, fluix crystal.  if you throw your hands, the crafting, through МЕ does not work, single version 2 0 4 5

**Comment by lexx1191** [comment]
yet the only way to craft through advanced circuit

**Comment by Sarodinian** [comment]
My AE2 system is having issues with this as well. Have a Lapotron Crystal recipe for my AE2 system. Crafting stalls when it is called. AE2 ME interface directly on the assembler, nothing gets pushed in. Interface on a buffer chest, with conveyor pulling into the assembler, interface pushes into the chest, assembler does not accept the circuits. Interface to Buffer chest, conveyor to pipe, then fed to assembler, gets the same result. Raw crystal goes in fine, circuits do not. Golem on Empty core does not take circuits from a buffer chest to the assembler either. Changing the HV circuit type (Advanced Circuit vs Processor Assembly vs Nanoprocessor) has no effect either. The only way to get the recipe to work seems to be to place the circuits inside the assembler myself.

**Comment by Mc-Flausch** [comment]
Today (Yesterday) I have had this happening to me too. Still does not work.

Moved my old GT Instance to a different Folder and downloaded a whole new instance of the Modpack through the Technic Launcher. Created a fresh new World for testing purposes.

Version is: 2.0.7.0
Still does not work.
Can't pipe nor conveyor at least 4 Circuits into an assembler. 
Tried Assembling Machines from LV up to IV. 
Those 4 don't want to get moved via pipe/ conduit etc.

Microprocessor 6854:32078 LV-Tier
Processor Assembly 6852:32703 HV-Tier
Nanoprocessor 6854:32082 HV-Tier
Bio Mainframe 6854:32120 UEV-Tier

Also I noticed, when inserting about 20 different circuits into a chest to get distributed via enderIO item conduits with round robin activated, my frames dropped from about 110 to 26 for an instance.
